charts/opentelemetry-demo/values.yaml

```diff
@@ -579,6 +579,7 @@ opentelemetry-collector:
       otlp:
         protocols:
           grpc:
+            endpoint: 0.0.0.0:4317          
```

Comment:
This is actually already the default value for the grpc object in the collector chart: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/22bfae673b305d73ad99815d38bc16aff6bec953/charts/opentelemetry-collector/values.yaml#L85

I actually think we can get away with removing the `grpc` section and rely on the default collector chart config.  @avillela can you try that out?

Reply:
Yeah, that did the trick. Removing it altogether caused it to get picked up from the collector chart. I think maybe because the original entry was:

```
otlp:
    protocols:
        grpc:
```

It probably overrode the inherited value. Config has been updated.